reddit_aita/models/aita_comments.sql

```diff
@@ -6,12 +6,6 @@ FROM {{ source('reddit_comments', '20*') }}
 WHERE subreddit = 'AmItheAsshole'
 AND _table_suffix > '19_'
 
-{%- if is_incremental() -%}
-{%- if execute -%}
```

Comment:
So I'm using this weird complex logic because one of these queries is much better for BigQuery:

- SELECT * FROM `*` WHERE _table_suffix < '2019_01'
- SELECT * FROM `*` WHERE _table_suffix < (SELECT date FROM ...)

One is a constant and will prune how much data is read. The other one is variable, and BigQuery doesn't optimize it as well.

I can get that as a constant by using run_query(). I have another option, but it doesn't work now (https://github.com/fishtown-analytics/dbt/issues/2136)